app/uk/gov/hmrc/eusubsidycompliancefrontend/services/EscService.scala

```diff
@@ -59,6 +59,17 @@ class EscService @Inject() (
         } yield ref
       }
 
+  def disableUndertaking(undertaking: Undertaking)(implicit hc: HeaderCarrier): Future[UndertakingRef] =
+    escConnector
```

Comment:
If we disable the undertaking we probably need to flush the cache here too. Refer to `updateUndertaking`.

Reply:
Urgh, I can't read clearly. I can see it now!